docs/Analyzer Configuration.md

```diff
@@ -325,7 +325,48 @@ Additionally, all the dataflow analysis based rules can be configured with a sin
 
 Option Name: `excluded_type_names_with_derived_types`
 
-Configurable Rules: [CA1303](https://docs.microsoft.com/visualstudio/code-quality/ca1303)
+Configurable Rules:
```

Comment:
Let me know if you want me to revert this change and do a separate PR for it. Basically, except for `CA2000` before this PR, this user option was already available for all the rules listed in `excluded_symbol_names` but the doc wasn't correct.

Side note, I have seen that most of the website docs are also missing this and need to be updated too.